tests/shared/drive-constraints.spec.js

```diff
@@ -490,29 +657,53 @@ describe('Shared: DriveConstraints', function() {
 
       it('should return false if the drive is not large enough and is a source drive', function() {
         m.chai.expect(constraints.isDriveValid(this.drive, {
-          path: path.join(this.mountpoint, 'rpi.img'),
-          size: 5000000000
+          path: path.join(this.mountpoint, path.join(__dirname, 'rpi.img')),
```

Comment:
Still there in a local checkout (commit a7341086a2e825f6b1ee016f51a0707c67c4c3ea) of the `unify-image-object` branch:
```bash
$ grep -n path\.join tests/shared/drive-constraints.spec.js 
321:            path: path.join(__dirname, 'rpi.img'),
352:            path: path.join(__dirname, 'rpi.img'),
383:            path: path.join(__dirname, 'rpi.img'),
418:            path: path.join(__dirname, 'rpi.img'),
449:            path: path.join(__dirname, 'rpi.img'),
480:            path: path.join(__dirname, 'rpi.img'),
511:        path: path.join(__dirname, 'rpi.img'),
551:        path: path.join(__dirname, 'rpi.img'),
566:        path: path.join(__dirname, 'rpi.img'),
581:        path: path.join(__dirname, 'rpi.img'),
596:        path: path.join(__dirname, 'rpi.img'),
605:        path: path.join(__dirname, 'rpi.img'),
660:          path: path.join(this.mountpoint, 'rpi.img'),
686:          path: path.join(this.mountpoint, 'rpi.img'),
720:          path: path.join(this.mountpoint, path.join(__dirname, 'rpi.img')),
746:          path: path.join(this.mountpoint, path.join(__dirname, 'rpi.img')),
```